.docker/entrypoint/spec/gemfile_generator_exec_spec.rb

```diff
@@ -8,13 +8,15 @@
   let(:generated_gemfile_path) { File.join(__dir__, 'data', 'Gemfile.generated') }
 
   subject(:generator) do
-    GemfileGeneratorExec.new(
+    generator = GemfileGeneratorExec.new(
       {
         default: default_gemfile_path,
         user: user_gemfile_path,
         generated: generated_gemfile_path
       }
     )
+    generator.debug = true
```

Comment:
Is this debugging code, or should `generator.debug` always be true? If so then we can modify the code to not check for the variable and simplify it to always run as debug.